.github/workflows/realease-on-pr-merge.yaml

```
@@ -39,30 +42,32 @@ jobs:
+          echo "# What's Changed" > release_notes.md
+          echo "" >> release notes.md
+          gh pr list --search "merged:>=$(git describe --tags --abbrev=0)" --json title,number,labels --jq '.[] | "* \(.title) (#\(.number))"' >> release_notes.md
       - name: Create GitHub Release
         uses: softprops/action-gh-release@v2
         with:
           tag_name: ${{ steps.bump_version.outputs.new_tag }}
           name: Release ${{ steps.bump_version.outputs.new_tag }}
-          body: |
-            Changes in this Release:
-            ${{ github.event.pull_request.body }}
+          body_path: release_notes.md
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)